index.js

```diff
@@ -1,52 +1,49 @@
 // Objects and Functions
+
+//  Создать объект car, добавить к нему свойство color со значением 'черный'
+// Изменить свойство color объекта car на 'зеленый'
+// В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
 let car = {
-    color: 'black',
-    power: showPower()
+  color: "black",
+  power: showPower(),
 };
 function showPower() {
-    return '100ls';
-    console.log('100ls');
+  console.log("100hp");
 }
-    car.color = 'green';
-
-
-function fruitCount(apples, pies) {
-    console.log(apples+pies);
-    return apples+pies;
+car.color = "green";
 
+// На склад принимают груши и яблоки,
+// напишите функцию, которая возвращает результат сложения количества принятых груш и яблок
+function countingFruits(apples, pies) {
+  return apples + pies;
 }
-fruitCount(2,2);
-
-
-function nameTerminal(name) {
-    if(name==="Egor"){
-        alert('privet'+name);
-    }
-    else{
-        alert('net imeni');
-    }
+fruitCount(2, 2);
+
+// В терминале оплаты сохранено ваше имя, напишите функцию
+// для определения имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
+function showNameTerminal(name) {
+  if (name === "Egor") {
+    alert("privet" + name);
+  } else {
+    alert("net imeni");
```

Comment:
Please use console.log() instead of alerts. Also fix the whole file.

I mentioned it here: https://github.com/egorRasolko/epam-js/pull/2#discussion_r850628765